### PR TITLE
fix: Prevent `rw graphiql setup` from breaking with encryptSession error

### LINKED ISSUE #9581: [Bug?]: `yarn rw setup graphiql dbAuth` fails on v6.4.0
### What's not working?

Example:

```
$ yarn rw setup graphiql dbAuth --id 1
✖ encryptSession is not a function
◼ Generating file in src/lib/generateGraphiQLHeader.{js,ts}...
◼ Importing generated headers into createGraphQLHandler
◼ Installing packages...
encryptSession is not a function
```

This used to work on v6.3.3, and broke when `encryptSession` was introduced, in c80df9b30ffbfbf1b9b75af92938a9941215a075.

### How do we reproduce the bug?

Run `yarn rw setup graphiql dbAuth --id 1`.

### What's your environment? (If it applies)

MacOS M1

### Are you interested in working on this?

- [X] I'm interested in working on this

### PR DESCRIPTION
**Pull request by staticshock**
Fixes #9581

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -7,7 +7,7 @@ root = true
 [*]
 end_of_line = lf
 insert_final_newline = true
-charset = utf8
+charset = utf-8
```
Comment (staticshock) [inline .editorconfig:10]: Err, this is unrelated, but didn't seem worth its own PR. Let me know if y'all disagree and I'll turn it into a separate PR.

**Comment by dac09** [comment]
Thanks for the PR @staticshock! Looking at it now :)

**Comment by dac09** [comment]
Thanks @staticshock - setup working correctly, however I think we also need the same change in `packages/studio/api/lib/authProviderEncoders/dbAuthEncoder.ts` 

Are you able to verify that the session is being impersonated in studio when you run `yarn rw experimental studio`?

**Comment by dac09** [comment]
Applied changes to studio as well, thanks @staticshock for the PR and the well written (and analysed) issue ✌️

**Review by dac09 — APPROVED** [review]

**Comment by staticshock** [comment]
Thanks for the quick merge!

**Pull request merged**